unit-tests/func/frame-filter/test-l500-frame-filter.py

```diff
@@ -0,0 +1,60 @@
+import pyrealsense2 as rs, test
```

Comment:
Copyright notice is missing.

```
#test:device L500*
```

`frame-filter` doesn't need to be in both the filename and directory name.
But why is it called a "frame filter"? Do you foresee other tests in here?
Right now functional tests are grouped by feature ("alt-ir", "fg"). Why not put this in an `ir/` folder? Maybe even with `l500/` under it?

Reply:
👍